CalibCalorimetry/EcalSRTools/test/produceDccWeights.py

```diff
@@ -15,14 +15,17 @@
     firstValid = cms.vuint32(1)
 )
 
+process.load("Geometry.HcalCommonData.hcalDDDSimConstants_cff")
+process.load("Geometry.HcalCommonData.hcalDDDRecConstants_cfi")
+
 # Conditions:
 #
 # a) Getting hardcoded conditions the same used for standard digitization:
 #process.load("CalibCalorimetry.EcalTrivialCondModules.EcalTrivialCondRetriever_cfi")
 #process.EcalTrivialConditionRetriever.producedEcalIntercalibConstants = True
 # b) Getting conditions through frontier interface:
 process.load('Configuration/StandardSequences/FrontierConditions_GlobalTag_cff')
-process.GlobalTag.globaltag = "CRAFT09_R_V4::All"
+process.GlobalTag.globaltag = "113X_dataRun3_HLT_v3"
```

Comment:
this is under test, so you can have whatever you would like here, but wouldn't it be easer to use the auto cond key instead of a specific GT?